src/reports/budgetingperf.cpp

```diff
@@ -26,7 +26,12 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #include "model/Model_Budget.h"
 #include "model/Model_Category.h"
 #include "reports/mmDateRange.h"
-
+#include <string>
+namespace tags
+{
+static const wxString TABLE_CELL_LINK = R"(<a href="%s" target="_blank">%s</a>)";
```

Comment:
Think we already have mmHTMLBuilder::addTableCellLink is it posible to re-use this in some way?

Reply:
I can try to reuse that, but I still need to have the TABLE_CELL_LINK_RED.

Reply:
Perhaps include TABLE_CELL_LINK_RED in mmHTMLBuilder also? Or add a colour option to addTableCellLink? Makes it usable elsewhere.

Reply:
since that cell contain a span:
```
hb.startSpan((budgetDeductMonthly && estimate != 0 && round(estimateTotal[12] / budgetStats[catID][12] * 100) / 100 > 1)
                 ? hb.getFormattedLink("red",editBudgetEntry,estimateVal)
                 : hb.getFormattedLink("", editBudgetEntry, estimateVal),
             wxString::Format(" style='text-align:right;%s' nowrap", ""));
hb.endSpan();
```
I cannot use the existent `addTableCellLink`. 
I create another method `getFormattedLink` in mmHTMLBuilder and a new TABLE_CELL_LINK_COLOR that can manage color.


```
static const wxString TABLE_CELL_LINK = R"(<a href="%s" target="_blank">%s</a>)";
static const wxString TABLE_CELL_LINK_COLOR = R"(<a style="color: %s;" href="%s" target="_blank">%s</a>)";
```

should be flexible and reusable

Reply:
@whalley  thank  you very much for accepting my proposal